cpp/src/arrow/compute/kernels/scalar_string_test.cc

```diff
@@ -19,6 +19,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include "arrow/compute/exec.h"
```

Comment:
Can you keep the includes grouped together?

Reply:
done